Firewall: changed file path in getNfsPorts to Debians path

hannes427 opened the pull request:
The file /etc/sysconfig/nfs does not exist in Debian. /usr/sbin/nfsconf outputs the value of the requested parameter (e.g. port of nfsd) or exits with exit-code 1 if the requested parameter is not set (--> the default value is used)

Bugfix FreePBX/issue-tracker#645

hannes427 committed:
Firewall: changed file path in getNfsPorts to Debians path
The file /etc/sysconfig/nfs does not exist in Debian. /usr/sbin/nfsconf
outputs the value of the requested parameter (e.g. port of nfsd) or
exits with exit-code 1 if the requested parameter is not set (--> the
default value is used)

Bugfix FreePBX/issue-tracker#645

ramarajan222 commented on Services.class.php:
```diff
@@ -612,34 +612,39 @@ private function getSvc_nfs() {
-		}
+		exec("/usr/sbin/nfsconf --get nfsd port", $output_nfsd, $output_nfsd_rc);
+        if ($output_nfsd_rc == 0) {
+            $nfs = $output_nfsd[0];
```
Hi @hannes427 Can you correct the typo please, change $nfs to $nfsd. 
 
`$nfsd = $output_nfsd[0];`

hannes427 replied:
Yes i can do it but this is my first Pull request so i have to ask: How should i do it? Do i have to edit the file Locally, push it to my own repo and do a second pull request or is there anouther way?

Edit: I think i found the answer...am i right?

hannes427 committed:
The file /etc/sysconfig/nfs does not exist in Debian. /usr/sbin/nfsconf
outputs the value of the requested parameter (e.g. port of nfsd) or
exits with exit-code 1 if the requested parameter is not set (--> the
default value is used)
Fixed typo in line 628

Bugfix FreePBX/issue-tracker#645

hannes427 committed:
Merge branch 'FreePBX:release/17.0' into clean-issue-645
